aviary/subsystems/propulsion/propulsion_builder.py

```diff
@@ -61,14 +61,12 @@ def report(self, prob, reports_folder, **kwargs):
         propulsion_outputs = [Aircraft.Propulsion.TOTAL_NUM_ENGINES,
                               Aircraft.Propulsion.TOTAL_SCALED_SLS_THRUST]
 
-        engine_outputs = [Aircraft.Engine.NUM_ENGINES,
-                          Aircraft.Engine.SCALE_FACTOR,
-                          Aircraft.Engine.SCALED_SLS_THRUST]
-
         with open(filepath, mode='w') as f:
-            f.write('# PROPULSION')
+            f.write('# Propulsion')
             write_markdown_variable_table(f, prob, propulsion_outputs, self.meta_data)
-            f.write('\n## ENGINES')
-            for engine in prob.aviary_inputs.get_val('engine_models'):
-                f.write(f'\n### {engine.name}')
-                write_markdown_variable_table(f, engine, engine_outputs, self.meta_data)
+            f.write('\n## Engines')
+
+        # each engine can append to this file
+        kwargs['meta_data'] = self.meta_data
+        for engine in prob.aviary_inputs.get_val('engine_models'):
```

Comment:
This won't put in duplicates, right?
(a conventional twin engine aircraft would only have one entry)

Reply:
Yeah, this is looping through each unique _type_ of engine